ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -200,15 +200,18 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-private-shared-vpc-phz-sts-f28
+- as: aws-ipi-private-shared-vpc-phz-sts-canary-f28
   cron: 32 19 16 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       EXTRACT_MANIFEST_INCLUDED: "true"
+      MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "mcpfoo", "mcp_node_num": "1", "mcp_node_label":
+        "node-role.kubernetes.io/worker="}]'
+      PAUSED_MCP_NAME: mcpfoo
```

Comment:
btw, @jiajliu should we add `FILTERS_ADDITIONAL: ~TELCO&` for these canary upgrade jobs?

Reply:
From the last test [result](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/55328/rehearse-55328-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-upgrade-from-stable-4.16-baremetalds-ipi-ovn-ipv4-fips-canary-f28/1828295103819550720/artifacts/baremetalds-ipi-ovn-ipv4-fips-canary-f28/cucushift-upgrade-eus-unpause-worker-mcp/build-log.txt), it seems no worker-pao mcp. 

```
all observed mcps: 
master mcpfoo worker
```
so it looks like no need.

Reply:
Good point, so once we saw `worker-pao` mcp, we should set the FILTER, right?

Reply:
I'm not sure how the PAO cases selected in different upgrade jobs, but from the description in the [pr](https://github.com/openshift/release/pull/52588), it should be. 
```
We need to skip PAO test case during EUS upgrade, don't expect additional mcp worker-pao during upgrade
```